setup.cfg

```diff
@@ -1,5 +1,6 @@
 [tool:pytest]
-exclude=test_(extractor_ctio_planetary_nebula|astrometry|mcmc|multispectra)
+# exclude=test_(extractor_ctio_planetary_nebula|astrometry|mcmc|multispectra)
+addopts = --ignore=tests/test_astrometry.py --ignore=tests/test_mcmc.py  --ignore=tests/test_extractor_ctio_planetary_nebula.py
```

Comment:
Why are you ignoring broken tests rather than deleting them (or renaming them so pytest doesn't find them)? It looks like we mark test_astrometry.py as broken in our branch and skip it.

Reply:
Because the doctests work fine on my laptop but I don't understand why in the github workflow they fail all. I still hope to find a solution one day so I don't delete the file...